### DISCUSSION TITLE
[@types/node] missing error codes for dns promises

### DISCUSSION THREAD

**Discussion by polomsky** [DefinitelyTyped/DefinitelyTyped#68750] [Issues with a @types package] [answered]
Based on the [documentation](https://nodejs.org/docs/latest-v18.x/api/dns.html#error-codes)  at least Node 18 (did not checked further) supports constants like NODATA for dns promises.

> The dnsPromises API also exports the above error codes, e.g., dnsPromises.NODATA.

So these lines should be working

```typescript
import * as dnsPromises1 from 'dns/promises';
import {promises as dnsPromises2} from 'dns';

const err1 = dnsPromises1.NODATA;
const err2 = dnsPromises2.NODATA;

```

[a comment by an automated account is not shown]

**Comment by ghost** [comment, marked as answer]
Added in PR #68752

**Reply by peterblazejewicz** [reply]
correction: it's a WIP, not added yet

**Reply by ghost** [reply]
It's merged now, I think it can be marked as answered now
